INFO: Installing PyCall v0.4.7
INFO: Package database updated
ERROR: TimeoutAsyncWork not defined
 in include at ./boot.jl:245
 in include_from_node1 at ./loading.jl:128
 in include at ./boot.jl:245
 in include_from_node1 at ./loading.jl:128
 in reload_path at loading.jl:152
 in _require at loading.jl:67
 in require at loading.jl:51
 in include at ./boot.jl:245
 in include_from_node1 at loading.jl:128
 in process_options at ./client.jl:285
 in _start at ./client.jl:354
while loading /home/idunning/pkgtest/.julia/v0.3/PyCall/src/gui.jl, in expression starting on line 127
while loading /home/idunning/pkgtest/.julia/v0.3/PyCall/src/PyCall.jl, in expression starting on line 509
while loading /home/idunning/pkgtest/.julia/v0.3/PyCall/testusing.jl, in expression starting on line 1
INFO: Package database updated
